src/pages/routes.tsx

```diff
@@ -1,14 +1,14 @@
 import { Routes, Route } from 'react-router-dom';
 import { ProfilePage } from './profile';
 import { HomePage } from './home';
-import { EditProfilePage } from './edit-profile';
+import { EditUserProfile } from 'features/editUserProfile';
```

Comment:
Feature is not a page. You should combine different features inside of a page and export page.

Reply:
fixed